### DISCUSSION TITLE
not responding

### DISCUSSION THREAD

**Discussion by snake-sea** [ciderapp/Apple-Music-Electron#408] [Q&A]
Click no response.Unable to access the software

**Comment by quacksire** [comment]
Have you downloaded it and... ran the executable?

**Comment by snake-sea** [comment]
I have doownloaded but I don't kown how to run executable.Just first to use.Can you tell me how to run? Really THanks




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Apple-Music-Electron/Apple-Music-Electron"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年12月4日(星期六) 凌晨1:02
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Apple-Music-Electron/Apple-Music-Electron] not responding (Discussion #408)





 
Have you downloaded it and... ran the executable?
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by quacksire** [comment]
Click it.
